app/src/main/java/com/onboarding/calculatorkotlin/mvp/model/CalculatorModel.kt

```diff
@@ -45,18 +49,83 @@ class CalculatorModel : CalculatorContract.Model {
     }
 
     override fun getLastModified(): String {
-        return if (secondOperand.isNotEmpty()) {
-            secondOperand
-        } else if (operator.isNotEmpty()) {
-            operator
-        } else {
-            firstOperand
+        return when {
+            secondOperand.isNotEmpty() -> {
+                secondOperand
+            }
+            operator.isNotEmpty() -> {
+                operator
+            }
+            else -> {
+                firstOperand
+            }
+        }
+    }
+
+    private fun operandEnabled(operand: String): Boolean {
+        return (operand != EMPTY_STRING && operand != SUB)
```

Comment:
here you can simplify more... like this 
private fun operandEnabled(operand: String) = operand != EMPTY_STRING && operand != SUB

Reply:
Solved!